src/platforms/node/guides/gcp-functions/index.mdx

```diff
@@ -4,44 +4,69 @@ redirect_from:
   - /platforms/node/gcp_functions/
 ---
 
-Add `@sentry/node` as a dependency:
+_(New in version 5.26.0)_
 
-```bash {tabTitle:npm}
-npm install --save @sentry/node
-```
+Add `@sentry/serverless` as a dependency to `package.json`:
```

Comment:
Why we use `package.json` instructions here, but `npm install` in AWSLambda? They should be consistent

Reply:
GCP pulls in npm modules from package.json without having to install Sentry.